cas/incident-report.md

```diff
@@ -174,6 +176,18 @@ Create a new Bugzilla issue by filling out [this form](https://bugzilla.mozilla.
 - The "Summary" field in Bugzilla (i.e., "Subject line") MUST begin with the CA Owner’s name, followed by a colon, and a brief title that highlights the type of incident being reported (e.g., "EXAMPLE CA OWNER: Incorrect Subject RDN Encoding"). The CA Owner's name SHOULD match exactly with the CA Owner value in the CCADB.
 - The "Description" field MAY contain a Preliminary or Full Incident Report (copied and pasted from the corresponding Markdown template, as explained [below](#report-templates)).
 
+#### How do I submit a security-sensitive incident report?
+
+For additional context on what might be considered a "security-sensitive" incident, see [here](https://wiki.mozilla.org/CA/Vulnerability_Disclosure).
+
+Sensitive security incidents and vulnerabilities can be reported in Bugzilla by filling out [this form](https://bugzilla.mozilla.org/enter_bug.cgi?bug_type=task&component=CA%20Security%20Vulnerability&groups=ca-program-security&product=CA%20Program). Under the heading "People", be sure to add incident-reporting [at] ccadb [dot] org to the report's CC list.
+
+_**Under the heading "Security", make sure that the "CA Program Security" checkbox is ticked -- leaving all security boxes unchecked makes it a public bug.**_
+
+After opening a security-sensitive incident in Bugzilla, also email incident-reporting [at] ccadb [dot] org.
+
+Following the submission of a security-sensitive incident in Bugzilla, the corresponding CA Owner MUST file a public incident report once they have determined that the associated security risk is closed.
```

Comment:
Is it acceptable to instead convert the initial report into a public report? If yes, how do they go about doing so?

Reply:
Sure, I think we can add language that says the CA Owner can contact incident-reporting@ and request that the private bug be converted to public. @BenWilson-Mozilla, is that correct?

The defined approach was written to support a scenario where - if in responding to questions from root programs the CA Owner disclosed sensitive information that presented ongoing risk to its operations or staff and was not specifically limited to the scope or duration of the incident (e.g., the exact address where key material was maintained) then a new incident report could be created (rather than trying to redact or delete old comments.)

Reply:
That's correct.

Reply:
```suggestion
Following the submission of a security-sensitive incident in Bugzilla, and once the associated security risk is mitigated, the corresponding CA Owner MUST either: 
- file a new public incident report for the event, or
- send an email to incident-reporting [at] ccadb [dot] org and request the existing security-sensitive incident be converted to a public incident report.
```